dask/dataframe/tests/test_dataframe.py

```diff
@@ -1609,13 +1610,19 @@ def test_dataframe_quantile(method, expected, numeric_only):
 
         assert (result == expected[1]).all().all()
 
-        with assert_numeric_only_default_warning(numeric_only):
+        with warnings.catch_warnings(record=True):
+            warnings.filterwarnings("ignore", category=FutureWarning)
+            # pandas issues a warning with 1.5, but not 1.3
```

Comment:
Hmm, based on the comment here I don't quite understand this change as shouldn't be expecting a warning in 1.3

https://github.com/dask/dask/blob/e893d5333ffca172d4c0abc46c3058055c436b82/dask/dataframe/tests/test_dataframe.py#L1522-L1530

Maybe I'm missing something tough

Reply:
Yes, I made Dask warn below 1.5 here:

https://github.com/dask/dask/blob/db4761aefc04305b9f723262eb9d78948fec7d60/dask/dataframe/core.py#L152

See [this comment](https://github.com/dask/dask/pull/9960/files#r1109023514) for the reasoning.